extensions/tailscale/package.json

```diff
@@ -6,17 +6,18 @@
   "icon": "command-icon.png",
   "author": "samlinville",
   "owner": "tailscale",
-  "access": "public",
```

Comment:
Please don't change the order of fields here, I'd like for `access` to remain under `owner`